diff --git a/closure/defs.bzl b/closure/defs.bzl
--- a/closure/defs.bzl
+++ b/closure/defs.bzl
@@ -26,7 +26,6 @@
 load("//closure/templates:closure_templates_plugin.bzl", _closure_templates_plugin = "closure_templates_plugin")
 load("//closure/testing:closure_js_test.bzl", _closure_js_test = "closure_js_test")
 load("//closure/testing:phantomjs_test.bzl", _phantomjs_test = "phantomjs_test")
-load("//closure/testing:web_test_repositories.bzl", _setup_web_test_repositories = "setup_web_test_repositories")
 load("//closure:filegroup_external.bzl", _filegroup_external = "filegroup_external")
 load("//closure:webfiles/web_library.bzl", _web_library = "web_library")
 load("//closure:webfiles/web_library_external.bzl", _web_library_external = "web_library_external")
@@ -47,7 +46,6 @@
 closure_templates_plugin = _closure_templates_plugin
 closure_js_test = _closure_js_test
 phantomjs_test = _phantomjs_test
-setup_web_test_repositories = _setup_web_test_repositories
 filegroup_external = _filegroup_external
 web_library = _web_library
 web_library_external = _web_library_external
